src/Momento.Sdk/Internal/Utils.cs

```diff
@@ -125,4 +125,26 @@ public int GetHashCode(T obj)
     /// so comparisons operate on byte-array content instead of reference.
     /// </summary>
     public static StructuralEqualityComparer<byte[]> ByteArrayComparer = new();
+
+    /// <summary>
+    /// Thrown an exception if the end index is larger than the start index.
+    /// </summary>
+    /// <param name="startIndex">Starting inclusive index of operation.</param>
+    /// <param name="endIndex">Ending exclusive index of operation.</param>
+    /// <exception cref="Momento.Sdk.Exceptions.InvalidArgumentException"><paramref name="endIndex"/> is less than or equal to <paramref name="startIndex"/></exception>
+    public static void ValidateStartEndIndex(int? startIndex, int? endIndex)
```

Comment:
what happens if they are null?  I don't know how C# evaluates `null >= 0`?

Reply:
i almost wonder if you only want to call this if they're both not null?  in which case you don't need the signature to allow them to be nullable?

Reply:
Can test explicitly with `.HasValue` as described [here](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/nullable-value-types)

```csharp
if (startIndex.HasValue && endIndex.HasValue)
{
```

Reply:
👍